backend/src/apiserver/client/swf.go

```diff
@@ -41,13 +41,15 @@ func (swfClient *SwfClient) ScheduledWorkflow(namespace string) v1beta1.Schedule
 }
 
 // creates a new client for the Kubernetes ScheduledWorkflow CRD.
-func NewScheduledWorkflowClientOrFatal(initConnectionTimeout time.Duration) *SwfClient {
+func NewScheduledWorkflowClientOrFatal(initConnectionTimeout time.Duration, clientQPS float32, clientBurst int) *SwfClient {
```

Comment:
Is there an existing struct type we can use that includes clientQPS and clientBurst?
If not, shall we create a common struct type for our own use-case?

Using a struct as argument would be better for longer term maintenance, because it allows some arguments to be optional --- when we add more arguments, we don't need to change every test calling this function

Reply:
+1

Reply:
Unfortunately there is not, I created a new struct named `util.ClientParameters` to contain clientQPS and clientBurst.